src/components/aelin/ENSOrAddress.tsx

```diff
@@ -6,19 +6,21 @@ import { CellProps } from '@/src/components/pureStyledComponents/common/Table'
 import { ExternalLink as Wrapper } from '@/src/components/table/ExternalLink'
 import { ChainsValues } from '@/src/constants/chains'
 import { useEnsLookUpAddress } from '@/src/hooks/useEnsResolvers'
+import { useWeb3Connection } from '@/src/providers/web3ConnectionProvider'
 import { getExplorerUrl } from '@/src/utils/getExplorerUrl'
 import { shortenAddress } from '@/src/utils/string'
 
 interface Props extends CellProps {
   address: string
-  network: ChainsValues
+  network?: ChainsValues
```

Comment:
Why make this optional? In which scenario the network wont be defined?

Reply:
There is a scenario where it's not defined and that's why I'm making it optional: https://github.com/AelinXYZ/aelin-frontend-v2/pull/736/files#diff-4c5b2fa69c70ae9de95adcad7e088376ab5b081ac9da6f2e03c44707ed9eb42eR245